genai-perf/genai_perf/llm_inputs/llm_inputs.py

```diff
@@ -783,41 +771,6 @@ def _convert_generic_json_to_output_format(
 
         return output_json
 
-    @classmethod
-    def _convert_generic_json_to_openai_chat_completions_format(
-        cls,
-        dataset_json: Dict,
-        add_model_name: bool,
-        add_stream: bool,
-        extra_inputs: Dict,
-        output_tokens_mean: int,
-        output_tokens_stddev: int,
-        output_tokens_deterministic: bool,
-        model_name: list = [],
-        model_selection_strategy: ModelSelectionStrategy = ModelSelectionStrategy.ROUND_ROBIN,
-    ) -> Dict:
-        # TODO (TMA-1757): Implement a way to select a role for `text_input`
-        (
-            system_role_headers,
-            user_role_headers,
-            _,
-        ) = cls._determine_json_feature_roles(dataset_json)
```

Comment:
### WHY
Removed this from chat completions because genai-perf _only_ supports user role in the openai chat endpoint anyway. There's no way for users to set other roles such as system or assistant, and if we need to support other roles, then we likely need some abstraction around openai chat message to make it more easy to maintain and work with.

### FOLLOW UP WORK
I only removed this for chat completions endpoint in this PR, but other code needs to be cleaned as well because it makes no sense for other endpoints to have this (since this is chat completions specific code). For instance, completions endpoint does not support system, user, assistant roles anyway. Same for Trition endpoints but since we were moving too fast and was just copying the same code, the code lives everywhere.